packages/codecs-core/src/fixCodec.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { fixBytes } from './bytes';
+import { Codec } from './codec';
+import { NotEnoughBytesError } from './errors';
+
+/**
+ * Creates a fixed-size codec from a given codec.
+ *
+ * @param codec - The codec to wrap into a fixed-size codec.
+ * @param fixedBytes - The fixed number of bytes to read.
+ * @param description - A custom description for the codec.
+ */
+export function fixCodec<T, U extends T = T>(
+    codec: Codec<T, U>,
+    fixedBytes: number,
+    description?: string
+): Codec<T, U> {
+    return {
+        description: description ?? `fixed(${fixedBytes}, ${codec.description})`,
```

Comment:
How about we throw these out except in dev mode?

```diff
-         description: description ?? `fixed(${fixedBytes}, ${codec.description})`,
+         description: __DEV__
+             ? (description ?? `fixed(${fixedBytes}, ${codec.description})`)
+             : undefined,
```

Reply:
I don't understand the point of that change? All of these descriptions can be overridden by the end-user and provide a helpful way to debug which codec failed. What would be the benefit of stripping them? Would that even be tree-shaken?

Reply:
Gotcha! Yeah then I don't see why not. I'll make the changes.

Reply:
Although, the problem with that is it makes it impossible to use the description of a codec inside an error message which could be convenient.